src/main/java/duchess/command/ByeCommand.java

```diff
@@ -0,0 +1,30 @@
+package duchess.command;
+
+import duchess.main.Duchess;
+
+/**
+ * This class contains the logic to handle the bye command.
+ *
+ * @author Amos Tan
+ * @version CS2103T AY21/22 Semester 1
+ */
+
```

Comment:
Shouldn't this blank line be deleted? 😄 I think this may appear in other areas also.